aws.yaml

```
@@ -1,4 +1,101 @@
 - active: false
+  apps:
+    - app: chart-operator
+      version: 0.11.2
+      use-upgrade-force: true
+      changelogs:
+        - description: Remove CPU Limits
+          type: Changed
+          urls:
+            - https://github.com/giantswarm/chart-operator/pull/335
+    - app: coredns
+      version: 1.1.3
+      componentVersion: 1.6.5
```

```suggestion
      componentVersion: 1.6.5
      version: 1.1.3
```

I think we should order the fields.